catroid/src/lunaAndCat/java/org/catrobat/catroid/common/FlavoredConstants.java

```diff
@@ -25,15 +25,21 @@
 
 import android.os.Environment;
 
+import org.catrobat.catroid.CatroidApplication;
+
 import java.io.File;
 
 import static org.catrobat.catroid.common.Constants.MAIN_URL_HTTPS;
 
 public final class FlavoredConstants {
 
-	public static final File DEFAULT_ROOT_DIRECTORY = new File(
+	public static final File DEFAULT_ROOT_DIRECTORY = CatroidApplication.getAppContext().getFilesDir();
+
+	public static final File EXTERNAL_STORAGE_ROOT_DIRECTORY = new File(
```

Comment:
its probably best to define "Pocket Code", "Luna&Cat", etc once here in constance, and then use this constant for creating EXTERNAL_STORAGE_ROOT_DIRECTORY, or we will fall in the same trap as below again and again....